### DISCUSSION TITLE
Configure X minutes

### DISCUSSION THREAD

**Discussion by subhash-kashyap** [Vinzent03/obsidian-git#117] [Problems] [answered]
Hi, I have just setup the plugin, in my Windows machine and Mac. I would like to know how I could go about configuring the time to backup, Ideally I would like to backup whenever I make a change.

Here is what I did so far after I have installed the plugin in my Windows Obsidian 
1. Enabled it in the community plugins tab
2. Got a prompt saying something similar to git was not setup
3. Went to my vault where I have the .obsidian folder, did a git init, add, remote, and push 
4. Cloned in my Mac, pulled remote main
5. Checked to install git-obsidian, shows its already installed 
6. Made a few changes onto my notes in the Mac Obsidian
7. After sometime Checked windows Obsidian, changes dont reflect

**Comment by Vinzent03** [comment, marked as answer]
Sorry, I think you misunderstood how Git works. You take snapshots of the current vault state and "upload" it to GitHub, where it can be "downloaded" to apply the changes locally. Git is not made to be a real-time sync service. To "upload" (commit and push) the local changes, you have to run the "Create Backup" command in Obsidian.

**Reply by subhash-kashyap** [reply]
I took the snapshot and pushed it to Github, cloned it to the different devices. My intention is to use the feature of this plugin where it automatically takes vault backup and I wish to know how to configure it.
<img width="404" alt="image" src="https://user-images.githubusercontent.com/13069873/134465115-de252a4b-61bb-4e2e-9882-78bf5f8ad06f.png">

I cant seem to find the Create Backup command in Obsidian, could you help me out with that

**Reply by Vinzent03** [reply]
To enable the auto backup you have to go to the plugin's settings. That's in the left sidebar in the Obsidian settings. Select "Obsidian Git" and edit the duration between each backup.
![grafik](https://user-images.githubusercontent.com/63981639/134470056-4771703c-e0c6-4468-bc13-6dae5225f0de.png)
To call the command press `CTRL + P` and search for git. Or you press the command palette icon in the left sidebar.

**Reply by subhash-kashyap** [reply]
Yep, thanks that helps.
